docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -23,8 +23,13 @@ NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the
 
 * Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.
 * View detection alerts generated by a specific rule. To do this, go to *Detect* -> *Rules*, then select a rule name in the All rules table. The *Rules detail page* displays a comprehensive view of the rule's details, and alert details are displayed in the Alerts table beneath the Trend histogram. Alternatively, you can go to *Detect* -> *Alerts* -> *Manage rules*, then click on a rule name in the All rules table to view the alerts associated with the rule.
-* Visualize alerts by specific parameters by using the *Stack by* drop-down in the Trend histogram. For example, if you select `signal.rule.name`, the histogram displays the total alert counts by rule name.
-* Group alerts by specific parameters and view individual alert counts by using the *Stack by* drop-down in the Count table. For example, if you select `signal.rule.name`, the table displays the number of alerts generated by each rule.
+* Group alerts by specific parameters and view individual alert counts by using the *Stack by* drop-down in the *Count* table. For example, if you select `signal.rule.name`, the table displays the number of alerts generated by each rule.
+* Visualize alerts by specific parameters by using the *Stack by* drop-down in the *Trend* histogram. For example, if you select `signal.rule.name`, the histogram displays the total alert counts by rule name.
+* Hover over a value in the data grid to display the inline actions available, such as *Filter In*, *Filter Out*, and *Add to timeline investigation*. Click the expand icon to open a full context menu of options, including *Show top values* and *Copy to Clipboard*. The options available vary depending on the type of data.
```

Comment:
Just have one small nitpick/question: do we call the expand icon the **View details** button or is that a different button than the one that displays when you hover over a value in the data grid?

Reply:
@nastasha-solomon They're two different UI elements, with similar-looking icons. The `View details` button opens the flyout with all of the data from the selected alert, while the expand icon that appears upon hovering (which unfortunately doesn't have a name) opens a context menu of actions you can perform on the selected value.